src/auto_scheduler/utils.h

```diff
@@ -253,6 +254,16 @@ inline std::string Chars(const char& str, int times) {
   return ret.str();
 }
 
+/*! \brief Print the time elapsed */
+inline void PrintTimeElapsed(std::chrono::time_point<std::chrono::high_resolution_clock> t_begin,
```

Comment:
I think it is a little strange to pass the `t_begin` and elapse the duration time. I would suggest we pass the `duration` to `PrintElapsedTime` (this name should be better, but current fn name is ok too).

i.e. 
```cpp
double duration = std::chrono::duration_cast<std::chrono::duration<double>>(
                        std::chrono::high_resolution_clock::now() - t_begin)
                        .count();
inline void PrintElapsedTime(double duration, int verbose) {
  StdCout(verbose) << "Time elapsed: " << std::fixed << std::setprecision(2) << duration << " s"
                   << std::endl;
}
```

Reply:
I feel that
`
double duration = std::chrono::duration_cast<std::chrono::duration<double>>(
                        std::chrono::high_resolution_clock::now() - t_begin)
                        .count();
`
is too long and makes the readbility worse. This is the reason for creating `PrintElapsedTime`

Reply:
How about we define one macro to help the readability?

```cpp
#define duration(x) std::chrono::duration_cast<std::chrono::duration<double>>(x).count()
#define now() std::chrono::high_resolution_clock::now()

auto elapsed_time = duration(now() - t_begin)

PrintElapsedTime(elapsed_time, verbose);

inline void PrintElapsedTime(double elapsed_time, int verbose) {
  StdCout(verbose) << "Time elapsed: " << std::fixed << std::setprecision(2) << elapsed_time << " s"
                   << std::endl;
}
```